doc/uml/TcpServerClass.uml

```diff
@@ -1,6 +1,9 @@
 @startuml
+Interface TcpServer
+FactoryTcpServer ..> TcpServer
+TcpServer ..|> TcpServerA
 TcpServerBase <|-- TcpServerCmd
 TcpServerBase <|-- TcpServerTel
-TcpServer *-- TcpServerCmd
-TcpServer *-- TcpServerTel
+TcpServerA *-- TcpServerCmd
+TcpServerA *-- TcpServerTel
```

Comment:
The class diagram should be:

"""
@startuml
TcpServerBase <|-- TcpServerCmd
TcpServerBase <|-- TcpServerTel
TcpServer *-- TcpServerCmd
TcpServer *-- TcpServerTel
FactoryTcpServer --> TcpServer
@enduml
"""

Reply:
Done. Thanks !